doc/fundamentals/getting_data_out_of_pachyderm.md

```diff
@@ -51,9 +51,62 @@ REPO                ID                                 PARENT
 edges               026536b547a44a8daa2db9d25bf88b79   754542b89c1c47a5b657e60381c06c71   3 minutes ago       Less than a second   133.6 KiB
 ```
 
-## Exporting data via `egress`
-
-In addition to getting data out of Pachyderm with `pachctl get file`, you can add an optional `egress` field to your [pipeline specification](../reference/pipeline_spec.html).  `egress` allows you to push the results of a Pipeline to an external data store such as S3, Google Cloud Storage or Azure Blob Storage. Data will be pushed after the user code has finished running but before the job is marked as successful.
+## Exporting data by using `egress`
+
+In addition to getting data out of Pachyderm by using
+`pachctl get file`, you can add an optional `egress` field
+to your [pipeline specification](../reference/pipeline_spec.html).
+`egress` enables you to push the results of a pipeline to an
+external datastore such as Amazon S3, Google Cloud Storage, or
+Azure Blob Storage. After the user code has finished running but
+before the job is marked as successful, Pachyderm pushes the data
+to the specified destination.
+
+You can specify the following `egress` protocols for the
+corresponding storage:
+
+<style type="text/css">
```

Comment:
whats with all this html boilerplate? It shows up here pretty badly: https://github.com/svekars/pachyderm/blob/1718/doc/fundamentals/getting_data_out_of_pachyderm.md

Does this render differently on RTD? Ideally, the docs should look ok in MD on GH as well.

Reply:
It renders nicely in readthedocs:

<img width="1680" alt="Screen Shot 2019-06-20 at 4 37 34 PM" src="https://user-images.githubusercontent.com/5317992/59887889-f3ac6c00-9379-11e9-861b-1fcf27441ef1.png">

Bummer, I'll have to try to find a way it on GitHub. Md tables are not great and I've seen many issues with .md table markup on readthedocs. I'll see if I can fix the html code so it's not visible on github.

Reply:
I struggled with this, too.  I never tried loading the css in a separate file, because I couldn't figure out how to get github to refer to the file so it'd render right.

Reply:
Alright. FIxed by changing to .rst for now. Will think about a long-term solution as well.